### PR TITLE
revisions of preregprivate

### PR DESCRIPTION
**Pull request by evomellor**
This is the page that will describe the process of the prereg challenge during the private launch period. During late December or early Jan it will replace cos.io/prereg as the main Prereg Challenge page.

### CODE REVIEW

**Comment by sloria** [comment]
I am getting a 404 page when I visit this link:

cos___get_involved_with_open_science

**Comment by sloria** [comment]
Smart quotes should not be used for `href` attributes. On Mac, you can disable smart quotes in Preferences > Keyboard > Use smart quotes and dashes (uncheck).

**Comment by evomellor** [comment]
I just submitted a PR that should fix it, thanks for seeing that!

David Mellor, PhD https://osf.io/qthsf/
Project Manger, Preregistration Challenge https://cos.io/prereg, Center
for Open Science
The Open Science Framework https://osf.io is an open source scholarly
commons to manage the entire research workflow.
@EvoMellor https://twitter.com/@EvoMellor

**Comment by evomellor** [comment]
Thanks, just disabled smart quotes.

File: www/getlisted.mako [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    <!-- BEGIN CONTAINER -->
+    <div class="container min-height margin-top-50 col-md-12">
+        <div class="center" >
+        <h1>Journal Engagement with the Preregistration Challenge</h1><p>
+        <h2>A community effort to increase transparency in scientific research.</h2>
+<img src="/static/img/pics/pre-reg-flow-bold.png" alt="preregistration workflow" width="500px" class="margin-top-40">
+        </div>
+
+        <div class=" margin-top-30  margin-bottom-30">
+            <h2><strong>Journal eligibility for the <a href="https://cos.io/prereg">Preregistration Challenge</a></strong></h2>
```
Comment (sloria) [inline www/getlisted.mako:80]: Internal links should not have the domain, i.e. `href="/prereg/'`. This is what is causing the 404 errors (see previous comment).

File: www/getlisted.mako [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+<%def name="description()">Get more journals listed as eligible for the Preregistration Challenge.</%def>
+
+<%def name="navigation()">
+    ${self.navlinks('prereg')}
+</%def>
+
+<%def name="stylesheets()">
+
+    <!-- BEGIN PAGE LEVEL PLUGIN STYLES -->
+    <link href="../static/plugins/fancybox/source/jquery.fancybox.css" rel="stylesheet" />
```
Comment (sloria) [inline www/getlisted.mako:12]: What's the purpose of these stylesheets and the embedded styles below? Can these be removed? On first look, they don't seem to have an effect on the page, but I may be missing something.

File: www/getlisted.mako [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    <!-- END PAGE LEVEL PLUGIN STYLES -->
+
+</%def>
+
+
+<!-- BEGIN PAGE CONTAINER -->
+<div class="page-container pre-reg">
+
+    <!-- BEGIN CONTAINER -->
+    <div class="container min-height margin-top-50 col-md-12">
```
Comment (sloria) [inline www/getlisted.mako:72]: The `min-height` and `margin-top-50` classes don't appear to have any effect. Did you mean to do `style="margin-top: 50px`? @evomellor Can you run this locally and ensure that this page appears the way you intend it to?

File: www/getlisted.mako [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+    </div>
+</div>
+
+<!-- END PAGE CONTAINER -->
+<!-- BEGIN PAGE LEVEL JAVASCRIPTS(REQUIRED ONLY FOR CURRENT PAGE) -->
+<%def name="javascript_bottom()">
+
+
+    <script type="text/javascript" src="assets/plugins/fancybox/source/jquery.fancybox.pack.js"></script>
```
Comment (sloria) [inline www/getlisted.mako:109]: @erinspace Is this JS needed here?

File: www/preregprivate.mako [hunk 5]
```diff
@@ -0,0 +1,536 @@
+<%inherit file="_base.mako"/>
+<%def name="title()">Preregistration Challenge</%def>
+<%def name="description()">Information about the Preregistration Challenge.</%def>
+<%def name="navigation()">
+    ${self.navlinks('communities')}
+</%def>
+
+<%def name="stylesheets()">
+<!-- BEGIN PAGE LEVEL PLUGIN STYLES -->
+    <link href="../static/plugins/fancybox/source/jquery.fancybox.css" rel="stylesheet" />
```
Comment (sloria) [inline www/preregprivate.mako:10]: @erinspace Is this file necessary here?

File: www/preregprivate.mako [hunk 6]
```diff
@@ -0,0 +1,536 @@
...
+    <link href="/static/css/bootstrap-table.css" rel="stylesheet" >
+    <link href="//cdn-images.mailchimp.com/embedcode/classic-081711.css" rel="stylesheet" type="text/css">
```
Comment (sloria) [inline www/preregprivate.mako:12]: @erinspace Is this file necessary?

**Comment by sloria** [comment]
@erinspace Disregard my previous comments. I will look into those stylesheets and JS.

**Comment by sloria** [comment]
Changes made during merge. This is now live
